DEPLOY.md

```diff
@@ -177,6 +177,9 @@ In the following instructions, "X.Y.Z" is used to denote the version of Cypress
     - https://github.com/cypress-io/cypress-fiddle
     - https://github.com/cypress-io/cypress-example-piechopper
     - https://github.com/cypress-io/cypress-documentation
+20. Check if any test or example repositories have a branch for testing the features or fixes from the newly published version `x.y.z`. The branch should also be named `x.y.z`. Check all `cypress-test-*` and `cypress-example-*` repositories, and if there is a branch named `x.y.z`, merge it into `master`.
+    - [cypress-test-tiny](https://github.com/cypress-io/cypress-test-tiny), [cypress-test-nested-projects](https://github.com/cypress-io/cypress-test-nested-projects), [cypress-test-example-repos](https://github.com/cypress-io/cypress-test-example-repos), [cypress-test-node-versions](https://github.com/cypress-io/cypress-test-node-versions), [cypress-test-module-api](https://github.com/cypress-io/cypress-test-module-api), [cypress-test-ci-environments](https://github.com/cypress-io/cypress-test-ci-environments)
+    - [cypress-example-todomvc](https://github.com/cypress-io/cypress-example-todomvc), [cypress-example-todomvc-redux](https://github.com/cypress-io/cypress-example-todomvc-redux), [cypress-example-realworld](https://github.com/cypress-io/cypress-example-realworld), [cypress-example-recipes](https://github.com/cypress-io/cypress-example-recipes), [cypress-example-docker-compose](https://github.com/cypress-io/cypress-example-docker-compose), [cypress-example-api-testing](https://github.com/cypress-io/cypress-example-api-testing), [cypress-example-piechopper](https://github.com/cypress-io/cypress-example-piechopper), [cypress-documentation](https://github.com/cypress-io/cypress-documentation)
```

Comment:
Think this should be a fully bullet pointed list, this will be pretty hard to check through I think as a comma separated paragraph.